kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/DescrNameViewModel.java

```diff
@@ -97,32 +96,36 @@ public Set<ConceptEntity> findAllStatuses(ViewProperties viewProperties) {
     }
 
     public Set<ConceptEntity> findAllCaseSignificants(ViewProperties viewProperties) {
-
-
-        //FIXME after connect-a-thon put this query back
-//        IntIdSet caseSenseDescendents = viewProperties.calculator().descendentsOf(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE.nid());
-//        Set<ConceptEntity> allCaseDescendents = caseSenseDescendents.intStream()
-//                .mapToObj(caseNid -> (ConceptEntity) Entity.getFast(caseNid))
-//                .collect(Collectors.toSet());
-
-        return CASE_SIGNIFICANCE_OPTIONS;
+        IntIdSet caseSenseDescendents = viewProperties.calculator().descendentsOf(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE.nid());
```

Comment:
Can we add a convenient function in `dev.ikm.komet.kview.mvvm.model.DataHelper` ?
```java
public static Set<ConceptEntity> fetchDescendentsOfConcept(ViewProperties viewProperties, PublicId publicId) {
   // return a set.
}

```

Now code can simply call this common function to get a set of concepts. This will remove the following common calls:

- Case significants
- Modules
- Path
- Or any similar use case.

Reply:
Done, please review. Do you want to call the helper method directly from the controller/View ? or via ViewModel ? I assumed via viewModel class, but then we can call it from view as well. The Helper resides in VM. Let me know if you need the call sequence to change and make it directly.